lib/eslint.js

```diff
@@ -24,19 +24,10 @@ var estraverse = require("estraverse-fb"),
 // Helpers
 //------------------------------------------------------------------------------
 
-// Have to modify estraverse until TryStatement.handlers is removed
-estraverse.VisitorKeys.TryStatement = ["block", "handlers", "finalizer"];
-
 // TODO: Remove when estraverse is updated
-estraverse.Syntax.ExportNamedDeclaration = "ExportNamedDeclaration";
```

Comment:
I forgot to take this out last time I updated estraverse. Just a little housecleaning.